II.pass-tests/03-sum-numbers/add-numbers.js

```diff
@@ -1,4 +1 @@
-function addNumbers(numbers) {
-
-}
-module.exports = addNumbers;
\ No newline at end of file
+module.exports = numbers => numbers.reduce((a, b) => a + b);
```

Comment:
Nice 😎 There is 1 little edge case that might be problematic here though - what happens if you pass an empty array (`[]`) to this function? How could you fix it?

Reply:
module.exports = (numbers = [0]) => 
    numbers.length == 0 ? 0 : numbers.reduce((a, b) => a + b);
    expect(addAllnumbers()).toBe(0);//default value
    expect(addAllnumbers([])).toBe(0);

Reply:
Firstly a quick github tip: if you put 3 backticks (```) on the line above a "block" of code then another 3 backticks on the line below. Then it will format the code nicely - it makes it easier to read.

Yep that would work! Nice work :)

There is another way that I was thinking of, when using the `.reduce` method. If you pass a second argument, it will use that to "initialise" the *accumulator* (which is the `a` variable in this example). So in this example the first time the callback function is called, `a` will be 0 and `b` will be 9. Then the next iteration, `a` will be 9 and `b` will be 23, etc. This is very useful in some cases.

Reply:
thanks

Reply:
Sorry about that! I know it is hard to read. I was trying to figure out how to attach codes. Thanks for letting me know!